src/python/WMCore/ReqMgr/DataStructs/Request.py

```diff
@@ -70,8 +70,12 @@ def generateRequestName(request):
     current_time = time.strftime('%y%m%d_%H%M%S', time.localtime(time.time()))
     seconds = int(10000 * (time.time() % 1.0))
     request_string = request.get("RequestString", "")
+
     if request_string != "":
         request["RequestName"] = "%s_%s" % (request["Requestor"], request_string)
```

Comment:
Yes, pdmvserv would be enough.

> Or we can created the lexicon for RequestString and prevent getting '@'. I guess you prefer second option
Sounds good to me as well. But we can make it for the next deployment IMO, a) enforce a RequestString and b) prevent such chars (@)